src/FINDOLOGIC/Helpers/ResponseHelper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace FINDOLOGIC\Helpers;
+
+class ResponseHelper
+{
+    /**
+     * Gets a property from an object.
+     *
+     * @param mixed $obj
+     * @param string $property
+     * @param null|string $type Convert value to another type. Optional.
+     * @param bool $emptyValueIsAllowed If true, values that are 0 or 0.0 are allowed.
+     *
+     * @return mixed|null Returns the property value or null if it does not exist.
+     */
+    public static function getProperty($obj, $property, $type = null, $emptyValueIsAllowed = false)
```

Comment:
Would it be a good idea to check if `$type` is one of a limited number of permissible types, and providing constants for them?

Reply:
Thinking further about this, maybe the best solution is to make this method private and call it from several public methods for each of the permitted types, indicating the type in the name, e.g. something like `getStringProperty` and `getIntProperty`.